### DISCUSSION TITLE
Easier organization of voicebanks, like ¿folders/categories? users can create to organizise the voicebanks

### DISCUSSION THREAD

**Discussion by axialuvv** [stakira/OpenUtau#1365] [Ideas]
![image](https://github.com/user-attachments/assets/d42cc79b-8d8c-4156-99d9-bcc9ec2e78e1)
like "Classic/Favourites/Diffsingers", well allow the user to create their own one with a the name they want and add the voices there

**Comment by BagelHero** [comment]
This has been suggested and worked on a little, but it seems the verdict was that it got a little cluttered.
Personally, I hope to see a feature like this in the future too, so hopefully some changes can be made to the idea in the future to make it a bit cleaner.
